.gitlab-ci.yml

```diff
@@ -26,6 +26,35 @@ check_w3c_css:
     - python3 -m json.tool < result
     - exit $RES
 
+check_w3c_html:
+  stage: test
+  before_script:
+    # Takes about 1 minute to pull
+    - echo -e "section_start:`date +%s`:docker\r\e[0Kdocker pull"
+    - time docker pull mvr320/djcihtml:latest
+    - echo -e "section_end:`date +%s`:docker\r\e[0K"
+  image: docker:stable
+  services:
+    - docker:19-dind
+    - mariadb
+  variables:
+    MYSQL_ROOT_PASSWORD: password
+    DOCKER_HOST: tcp://docker:2375/
+    DOCKER_DRIVER: overlay2
+  script:
+  - |
+    docker run --privileged \
+      -v $CI_PROJECT_DIR:$CI_PROJECT_DIR \
+      --net=host \
+      -e "TERM=xterm-256color" -e "HOME=$HOME" -e "USER=$USER" -e "MARIADB_PORT_3306_TCP_ADDR=$MARIADB_PORT_3306_TCP_ADDR" -e "MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD" \
+      mvr320/djcihtml:latest /bin/bash -eo pipefail -c "umask 0002; cd $CI_PROJECT_DIR; script --return -qfc \"./gitlab/html.sh\" /dev/null | ts \"[%F %T]\" "
```

Comment:
Probably better to push this under the domjudge user. Also we can probably keep the source in domjudge-script?

Reply:
Yes when the output was accepted I was going to start making PRs to the gitlabci project assuming thats the source you wanted there

Reply:
This one:
https://github.com/DOMjudge/domjudge-packaging/pull/63